samples/metadata-statistics/cp.cpp

```diff
@@ -0,0 +1,63 @@
+/* 
+ * Copyright 2015 Intel(r) Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http ://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+/*
+ * File copying routine for use in samples.
+ */
+
+#include <cstdio>
+#include <string>
+#include <iostream>
+
+using namespace std;
+
+void copyFile(const string& srcName, const char *dstName)
```

Comment:
C++ allows copying file in 3 lines of code:
```c++
ifstream source(srcName, ios::binary);
ofstream dest(dstName, ios::binary);
dest << source.rdbuf();
```
And if so I wouldn't suggest moving this code to a separate file.